src/mon/OSDMonitor.cc

```diff
@@ -13095,6 +13095,101 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
     wait_for_commit(op, new Monitor::C_Command(mon, op, 0, rs,
 					      get_last_committed() + 1));
     return true;
+  } else if (prefix == "osd pool force-remove-snap") {
```

Comment:
Let's add a comment here explaining the usage and purpose of the two variants of this command.  It doesn't need to be very detailed, simply enough that a future reader knows what it's supposed to do.